nixos/modules/services/web-servers/keter/config.nix

```diff
@@ -0,0 +1,74 @@
+{ globalketerConfig, keterRoot, writeShellScriptBin,
+  writeTextFile, lib, bundleCfg, callPackage, coreutils
+  , keter, ...
+}:
+
+let
+  incoming = "${keterRoot}/incoming";
+
+
+  globalKeterConfigFile = writeTextFile {
+    name = "keter-config.yml";
+    text = (lib.generators.toYAML { } globalketerConfig);
+  };
+
+  # If things are expected to change often, put it in the bundle!
+  bundle = callPackage ./bundle.nix
+    (bundleCfg // { exePkg = executable; });
+
+  executable = writeShellScriptBin "${bundleCfg.executable}" ''
+    #!/usr/bin/env bash
```

Comment:
A shebang will be added for you by `writeShellScriptBin`, this is a noop

Reply:
I deleted this